src/guide/component-basics.md

```diff
@@ -97,7 +97,9 @@ app.component('blog-post', {
 app.mount('#blog-post-demo')
 ```
 
-A component can have as many props as you'd like and by default, any value can be passed to any prop. In the template above, you'll see that we can access this value on the component instance, just like with `data`.
+When a value is passed to a prop attribute, it becomes a property on that component instance. The value of that property is accessible within the template, just like any other property of the component instance.
```

Comment:
**nitpick**: we have two repeating `component instance` in this paragraph. While it's technically correct, it reads a bit weird, maybe we could replace the secods case with `any other component property`?